src/insert_sort.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+import timeit
+
+
+best_case = [x for x in range(0, 1000)]
+worst_case = best_case[::-1]
+
+
+def insert_sort(l):
+    """Sort a list by inserting values in order after comparing each value."""
+    if len(l) <= 1:
+        return
+    else:
```

Comment:
This else statement is unnecessary, since the if conditional will return out of the function when `True`. Whenever possible, as in this case, it is preferable to avoid indented blocks so your code looks a little cleaner.